### ISSUE TITLE
Dygraphs in loop in RMarkdown

### ISSUE THREAD

**Issue by gunjanthesystem** [rstudio/dygraphs#64]
Hi,

I am not able to produce dygraphs in for loop inside RMarkdown document, seems like an issue
(Similar: http://stackoverflow.com/questions/30509866/for-loop-over-dygraph-does-not-work-in-r/31117554#31117554) 

Kindly look into it and thanks for this package.

**Comment by jjallaire** [comment]
Yes, this is a limitation of htmlwidgets (upon which dygraphs is based). We
have a PR in htmlwidgets pending that should address this issue:
https://github.com/ramnathv/htmlwidgets/pull/110

**Comment by DrNickBailey** [comment]
Is this the same reason that zero plots are displayed when I "Source" some linear code in which there are a number of Dygraph plots created?

**Comment by jjallaire** [comment]
You need source with echo = TRUE in order for the dygraphs to be printed
during the source.

**Comment by yihui** [comment]
I have posted an answer to your question on SO: http://stackoverflow.com/a/35235069/559676
